src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry.Test/packages.config

```diff
@@ -3,21 +3,21 @@
   <package id="Hyak.Common" version="1.0.3" targetFramework="net45" />
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.ContainerRegistry" version="1.2.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.ContainerRegistry" version="2.0.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Storage" version="6.5.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Test.Framework" version="1.0.6179.26854-prerelease" targetFramework="net45" />
-  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.7.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.8.1" targetFramework="net452" />
   <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
   <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.8" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.7" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="2.2.12" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.6.0" targetFramework="net45" />
-  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.10" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="2.3.0" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.7.2" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
```

Comment:
@Jinhuafei this can be reverted back to 6.0.8

Reply:
When updating Microsoft.Azure.Management.ContainerRegistry nupkg, it also updated all its dependencies. Why do we want to undo this? I know we have a global nupkg override, but it's too troublesome to manually maintain this. Plus this doesn't reflect the real dependency of the project.

Reply:
@Jinhuafei if you want to avoid these additional references from being added every time you update your management package, inside of the NuGet Package Manager in VS navigate to Options > Install and Update Options > Dependency behavior, and choose the "Ignore Dependencies" option. This will only install your management package and none of its dependencies.